src/rust/engine/process_execution/sandboxer/src/lib.rs

```diff
@@ -101,14 +107,18 @@ impl SandboxerService {
         ensure_socket_file_removed(&self.socket_path)?;
         // Wait a beat, to let any previous sandboxer process notice this removal and exit.
         // There's no great harm in leaving a stray sandboxer process running, it will
-        // receive no requests (since the socket file was pulled out from under it) so eventually
-        // it'll shut down due to idleness. But it's nicer if we can let it go right away.
+        // receive no new connections (since the socket file was pulled out from under it)
```

Comment:
Clarify that unix domain sockets are inode based, so deleting the file prevents new connections but doesn't prevent existing connections from functioning.